/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `wandb-entity` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `eval-checkpoint` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `target-kl` is annotated with type `<class 'float'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: florin-barbisch. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/wandb/run-20241223_000751-72rqrevx
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ALE/SpaceInvaders-v5__skip_frames_2__1__20241223_000744
wandb: ‚≠êÔ∏è View project at https://wandb.ai/florin-barbisch/cleanRL
wandb: üöÄ View run at https://wandb.ai/florin-barbisch/cleanRL/runs/72rqrevx
A.L.E: Arcade Learning Environment (version 0.10.1+unknown)
[Powered by Stella]
Exception ignored in: <function VectorEnv.__del__ at 0x7519aebf9120>
Traceback (most recent call last):
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 324, in __del__
    self.close()
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 222, in close
    self.close_extras(**kwargs)
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/sync_vector_env.py", line 305, in close_extras
    [env.close() for env in self.envs]
     ^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  [Previous line repeated 6 more times]
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/wandb/integration/gym/__init__.py", line 75, in close
    if not self.enabled:
           ^^^^^^^^^^^^
AttributeError: 'RecordVideo' object has no attribute 'enabled'
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `wandb-entity` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `eval-checkpoint` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `target-kl` is annotated with type `<class 'float'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: florin-barbisch. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/wandb/run-20241223_003301-ey1uc1ka
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ALE/SpaceInvaders-v5__skip_frames_4__1__20241223_003256
wandb: ‚≠êÔ∏è View project at https://wandb.ai/florin-barbisch/cleanRL
wandb: üöÄ View run at https://wandb.ai/florin-barbisch/cleanRL/runs/ey1uc1ka
A.L.E: Arcade Learning Environment (version 0.10.1+unknown)
[Powered by Stella]
Exception ignored in: <function VectorEnv.__del__ at 0x7a3ef11f5120>
Traceback (most recent call last):
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 324, in __del__
    self.close()
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 222, in close
    self.close_extras(**kwargs)
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/sync_vector_env.py", line 305, in close_extras
    [env.close() for env in self.envs]
     ^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  [Previous line repeated 6 more times]
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/wandb/integration/gym/__init__.py", line 75, in close
    if not self.enabled:
           ^^^^^^^^^^^^
AttributeError: 'RecordVideo' object has no attribute 'enabled'
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `wandb-entity` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `eval-checkpoint` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `target-kl` is annotated with type `<class 'float'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: florin-barbisch. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/wandb/run-20241223_010743-u9goxpug
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ALE/SpaceInvaders-v5__skip_frames_6__1__20241223_010739
wandb: ‚≠êÔ∏è View project at https://wandb.ai/florin-barbisch/cleanRL
wandb: üöÄ View run at https://wandb.ai/florin-barbisch/cleanRL/runs/u9goxpug
A.L.E: Arcade Learning Environment (version 0.10.1+unknown)
[Powered by Stella]
Exception ignored in: <function VectorEnv.__del__ at 0x75356ddf9120>
Traceback (most recent call last):
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 324, in __del__
    self.close()
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 222, in close
    self.close_extras(**kwargs)
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/sync_vector_env.py", line 305, in close_extras
    [env.close() for env in self.envs]
     ^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  [Previous line repeated 6 more times]
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/wandb/integration/gym/__init__.py", line 75, in close
    if not self.enabled:
           ^^^^^^^^^^^^
AttributeError: 'RecordVideo' object has no attribute 'enabled'
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `wandb-entity` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `eval-checkpoint` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `target-kl` is annotated with type `<class 'float'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: florin-barbisch. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/wandb/run-20241223_015137-ssy5uq72
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ALE/SpaceInvaders-v5__skip_frames_8__1__20241223_015133
wandb: ‚≠êÔ∏è View project at https://wandb.ai/florin-barbisch/cleanRL
wandb: üöÄ View run at https://wandb.ai/florin-barbisch/cleanRL/runs/ssy5uq72
A.L.E: Arcade Learning Environment (version 0.10.1+unknown)
[Powered by Stella]
Exception ignored in: <function VectorEnv.__del__ at 0x727c15ff1120>
Traceback (most recent call last):
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 324, in __del__
    self.close()
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 222, in close
    self.close_extras(**kwargs)
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/sync_vector_env.py", line 305, in close_extras
    [env.close() for env in self.envs]
     ^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  [Previous line repeated 6 more times]
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/wandb/integration/gym/__init__.py", line 75, in close
    if not self.enabled:
           ^^^^^^^^^^^^
AttributeError: 'RecordVideo' object has no attribute 'enabled'
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `wandb-entity` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `eval-checkpoint` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `target-kl` is annotated with type `<class 'float'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: florin-barbisch. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/wandb/run-20241223_024238-vjyl98h2
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ALE/SpaceInvaders-v5__skip_frames_16__1__20241223_024234
wandb: ‚≠êÔ∏è View project at https://wandb.ai/florin-barbisch/cleanRL
wandb: üöÄ View run at https://wandb.ai/florin-barbisch/cleanRL/runs/vjyl98h2
A.L.E: Arcade Learning Environment (version 0.10.1+unknown)
[Powered by Stella]
Exception ignored in: <function VectorEnv.__del__ at 0x72954f5f1120>
Traceback (most recent call last):
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 324, in __del__
    self.close()
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/vector_env.py", line 222, in close
    self.close_extras(**kwargs)
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/vector/sync_vector_env.py", line 305, in close_extras
    [env.close() for env in self.envs]
     ^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/gymnasium/core.py", line 336, in close
    return self.env.close()
           ^^^^^^^^^^^^^^^^
  [Previous line repeated 6 more times]
  File "/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/wandb/integration/gym/__init__.py", line 75, in close
    if not self.enabled:
           ^^^^^^^^^^^^
AttributeError: 'RecordVideo' object has no attribute 'enabled'
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `wandb-entity` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `eval-checkpoint` is annotated with type `<class 'str'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
/mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/.venv/lib/python3.12/site-packages/tyro/_parsers.py:344: UserWarning: The field `target-kl` is annotated with type `<class 'float'>`, but the default value `None` has type `<class 'NoneType'>`. We'll try to handle this gracefully, but it may cause unexpected behavior.
  warnings.warn(message)
wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: florin-barbisch. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.1
wandb: Run data is saved locally in /mnt/nas05/clusterdata01/home2/f.barbisch/rle-assginment/wandb/run-20241223_041808-qx63v1ik
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run ALE/SpaceInvaders-v5__skip_frames_32__1__20241223_041804
wandb: ‚≠êÔ∏è View project at https://wandb.ai/florin-barbisch/cleanRL
wandb: üöÄ View run at https://wandb.ai/florin-barbisch/cleanRL/runs/qx63v1ik
A.L.E: Arcade Learning Environment (version 0.10.1+unknown)
[Powered by Stella]
slurmstepd-server0101: error: *** JOB 59677 ON server0101 CANCELLED AT 2024-12-23T06:07:48 DUE TO TIME LIMIT ***
